packages/pdf-viewer/hooks/useScrollSaver.ts

```diff
@@ -1,15 +1,20 @@
-import { useRef, useEffect, MutableRefObject } from 'react';
-import { ScaledSize } from '../pdfSource';
+import { useRef, useEffect, MutableRefObject, useState } from 'react';
+import { ScaledSize, PdfData } from '../pdfSource';
```

Comment:
One thing about computers is that pretty much everything is "data" so creating a SomethingData class doesn't mean much. What is PdfData? Isn't it a wrapper around the PDF document? Then call it "PdfDocument". Also please create a separate file for it called PdfDocument.ts. Put ScaledSize in a separate types.ts file.